### PR TITLE
Add script to mirror one AzDO branch to another

### PR DESCRIPTION
**Pull request by wtgodbe**
Need to make a corresponding pipeline still

### CODE REVIEW

File: azure-pipelines-mirror-within-azdo.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Disable CI triggers, only called using Maestro
+trigger: none
+
+variables:
+  - group: Mirror-Credentials
+
+# Merges code from GitHub into internal branches in internal repos
```
Comment (dougbu) [inline azure-pipelines-mirror-within-azdo.yml:7]: Update comment

**Review by dougbu — COMMENTED** [review]
With one fix and assuming you've tested the YAML, this looks good, but I defer to @mmitche or similar for actual approval.
What is the plan after merging this❔ For example, will there be a new internal pipeline that can be kicked off manually, new configuration options in https://github.com/dotnet/versions/blob/main/Maestro/subscriptions.json, something I'm not thinking of&hellip;

**Comment by riarenas** [comment]
Should this script live in aspnetcore instead of arcade? I don't think we have widespread plans to support this through maestro (dotnet/versions) or as a widespread utility.

**Comment by wtgodbe** [comment]
> Should this yaml definition live in aspnetcore instead of arcade? I don't think we have widespread plans to support this through maestro (dotnet/versions) or as a widespread utility.

Why not have it be supported through dotnet/versions? My plan was to add an entry for it there after creating the pipeline

**Comment by riarenas** [comment]
I don't think maestro supports triggering changes based on an azdo repo, as it works with a GitHub webhook. Maybe @mmitche can confirm whether that's the case?

**Pull request closed without merging**